src/sql/src/parsers/alter_parser.rs

```diff
@@ -253,14 +262,60 @@ mod tests {
         }
     }
 
+    #[test]
+    fn test_parse_alter_change_column_type() {
+        let sql_1 = "ALTER TABLE my_metric_1 MODIFY COLUMN a STRING";
```

Comment:
Will the parser return an error if we also modify the column constraint? `ALTER TABLE my_metric_1 MODIFY COLUMN a STRING NOT NULL`

Reply:
I thought this would just ignore and not return an error